sections/semantics-textlevel.include

```diff
@@ -2957,11 +2957,32 @@
   value <a attr-value for="global/dir"><code>ltr</code></a> to specify a left-to-right override and with the value <a attr-value for="global/dir"><code>rtl</code></a> to
   specify a right-to-left override. The <a attr-value for="global/dir"><code>auto</code></a> value must not be specified.
 
-
   <p class="note">
     This element <a href="#bidireq">has rendering requirements involving the bidirectional algorithm</a>.
   </p>
 
+  <div class="example">
+      This example shows how the <{bdo}> element can be used to alter directionality formatting:
+
+      <xmp highlight="html">
+        <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+      </xmp>
+
+      <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+
+      This effect is achieved by applying the <code>rtl</code> attribute to the <{bdo}> element.
+  </div>
+
+  <div class="example">
+      If character encoding isn't appropriately defined, the characters of <abbr>RTL</abbr> languages
```

Comment:
appropriately defined should link somewhere, no? But see also #1273 where we might change this...

Reply:
Ah neat, I wasn't aware the spec recommended UTF-8!